### ISSUE TITLE
Cant extract, need help

### ISSUE THREAD

**Issue by Survivioboi** [Regalis11/Barotrauma#860]
When attempting to extract barotrauma i get this message ! C:\Users\bchic_000\Desktop\Barotrauma v0.8.2.1.zip: Cannot create C:\Users\bchic_000\Desktop\Barotrauma v0.8.2.1\Barotrauma.exe
  Access is denied.
This happened after I downloaded BTE, which caused Barotrauma.exe to disappear, I have deleted and re-
downloaded Barotrauma but this time it's not extracting

**Comment by BillCombs** [comment]
(Unrelated to the issue)
For some reason my last comment disappeared. I made a similar one to it, but after that the old one came back. I deleted the new one, and now the old one is gone again. Not sure what kind of glitch is that and if it's only on my side, but, just in case, i'm gonna leave this comment here and see if the old one comes back again.

Ah, looks like github is having problems https://status.github.com/messages
